ui/eureka/eureka.js

```diff
@@ -1966,6 +2430,9 @@ class EurekaTracker {
 
     document.getElementById('label-tracker').innerHTML = this.currentTracker;
 
+    if (this.zoneInfo.shortName == 'bozjasouthern')
```

Comment:
Yeah I think that'd be a cleaner way of doing this.

> 
> 
> In particular, doesn't this early out mean that castrum won't have a timer because it won't update?
> 
That's a good observation, I'll work on this.